<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RBF_To_C</class>
 <widget class="QWidget" name="RBF_To_C">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RBF_To_C</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0" colspan="8">
     <widget class="QPlainTextEdit" name="TextComment">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="plainText">
       <string>Add text comment to header file.</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="5">
     <widget class="QLineEdit" name="ArrayName">
      <property name="text">
       <string>FPGA_Firmware</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="labelArrayName">
      <property name="text">
       <string>Array name:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="5">
     <widget class="QLineEdit" name="Path_To_H">
      <property name="placeholderText">
       <string notr="true"/>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="OpenH_Button">
      <property name="text">
       <string>Select *.h file</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QRadioButton" name="IsFirstMSB">
      <property name="text">
       <string>First MSB</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="OpenRBF_Button">
      <property name="text">
       <string>Select *.rbf file</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QRadioButton" name="IsFirstLSB">
      <property name="text">
       <string>First LSB</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="5">
     <widget class="QLineEdit" name="Path_To_RBF">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string notr="true"/>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="labelColumnArray">
      <property name="text">
       <string>Count column</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="Convert_Button">
      <property name="text">
       <string>Convert!</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="4">
     <widget class="QCheckBox" name="IsAddDateTime">
      <property name="text">
       <string>Add date and time to *.h file</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="StatusPathRBF">
      <property name="maximumSize">
       <size>
        <width>8</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="7">
     <widget class="QLabel" name="status_label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QSpinBox" name="ColumnArray">
      <property name="value">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QLabel" name="StatusPathH">
      <property name="maximumSize">
       <size>
        <width>8</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="8">
     <widget class="QProgressBar" name="progressBar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>8</height>
       </size>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="format">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
